src/Compilers/CSharp/Test/Syntax/Parsing/LambdaAttributeParsingTests.cs

```diff
@@ -2305,10 +2306,7 @@ public void NullableType_Is_03()
         public void NullableType_Is_04()
         {
             string source = "_ = x is string ? [A] y => y : z";
-            UsingExpression(source,
-                // (1,1): error CS1073: Unexpected token '=>'
-                // _ = x is string ? [A] y => y : z
-                Diagnostic(ErrorCode.ERR_UnexpectedToken, "_ = x is string ? [A] y").WithArguments("=>").WithLocation(1, 1));
+            UsingExpression(source);
```

Comment:
the old parsing was just wrong.  This should have parsed according to the rules of the language.  The precedence rules say: https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/expressions.md#1115-conditional-operator

```g4
conditional_expression
    : null_coalescing_expression
    | null_coalescing_expression '?' expression ':' expression
    ;
```

So the true/false parts of a conditional can be full expressions (which includes lambdas).  So this should have correctly parsed before.  This is a fix.